modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
+
+When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.
```

Comment:
```suggestion
When you install Istio in your environment, you must configure Istio settings based on the Istio version that you install.
```

Reply:
I don't understand the purpose of this sentence; it seems to be stating the obvious, and in a bit of a confusing way.
If you install a particular version, don't the settings come with that version?
Are you trying to say that the audience needs to prepare its environment for the particular version of Istio it is installing? Or that different versions require different prep work?

Reply:
I now see that this is not about preparing but about how different Istio versions are installed in slightly different ways. Clarify this for the audience.

Reply:
Done! Thanks!